addons/website/views/website_templates.xml

```diff
@@ -140,7 +140,7 @@
         <attribute name="data-name">Footer</attribute>
         <!-- Background now controlled by css configuration, using color combinations -->
         <attribute name="t-attf-class" add="o_colored_level o_cc" remove="bg-light" separator=" "/>
-        <attribute name="t-cache">no_footer,no_copyright,website</attribute>
+        <attribute name="t-cache" add="website,main_object._name == 'website.page' and main_object"/>
```

Comment:
I doubt anyone will ever understand this line by just reading it, would you mind adding a quick comment about `footer_visible`? Thanks